sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsActionResult.java

```diff
@@ -12,7 +13,7 @@
 /**
  * The {@link TextAnalyticsActionResult} model.
```

Comment:
This JavaDoc is not particularly useful. The link is pointing to itself and documentation doesn't describe the purpose of this model.